velox/exec/Task.cpp

```diff
@@ -473,6 +473,59 @@ const std::string& Task::getOrCreateSpillDirectory() {
   return spillDirectory_;
 }
 
+Task::SplitStatus Task::getSplitStatus() {
+  SplitStatus splitStatus;
+
+  // Iterate drivers to
+  // 1. Collect running splits
+  // 2. Collect runningTableScanGroups: a map of <tableScan nodeId, [group ids]>
+  std::unordered_map<std::string, std::unordered_set<uint32_t>>
+      runningTableScanGroups;
+  for (auto driver : drivers_) {
```

Comment:
Most of the Task's members cannot be accessed w/o taking a lock first.
The Task's methods that access members are usually called after a lock is acquired and have `Locked` suffix.